model/schema/affectedGenomicModel.yaml

```diff
@@ -102,6 +119,19 @@ slots:
     range: string
     required: true
 
+  agm_secondary_ids:
+    description: Secondary IDs of a given allele
```

Comment:
description: Secondary IDs of a given AGM

Reply:
@chris-grove - Good spot.  Fixed along with another copy-paste error.